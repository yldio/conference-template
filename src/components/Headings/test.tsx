import React from 'react';
import { render } from '@testing-library/react';
import { H1, Variants } from './';

// add custom jest matchers from jest-dom
import '@testing-library/jest-dom/extend-expect';

describe('Button', () => {
  it('snapshot', () => {
    const { container } = render(<H1 variant={Variants.Dark}>Hello World</H1>);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render the given text', () => {
    const text = 'Hello World';
    const { getByText } = render(<H1>{text}</H1>);
    const content = getByText(text);

    expect(content).toBeInTheDocument();
  });
});
