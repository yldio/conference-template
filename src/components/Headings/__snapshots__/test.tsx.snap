// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot 1`] = `
<h1
  class="css-0"
>
  Hello World
</h1>
`;
